### ISSUE TITLE
Clear URLs breaks search in google sheets

### ISSUE THREAD

**Issue by nathantsoi** [ClearURLs/Addon#393]
When clearURLS is enabled, this dialog fails to appear in Google Sheets:

<img width="358" alt="Screenshot 2024-11-25 at 08 36 12" src="https://github.com/user-attachments/assets/a2179f6a-a5c4-4e57-b4aa-d7df0375d18f">

This error is shown in the console:
<img width="1463" alt="Screenshot 2024-11-25 at 08 37 11" src="https://github.com/user-attachments/assets/c9cb34af-48be-4d56-a270-607351deaee2">

**Comment by dreadbalaur** [comment]
Extra information: it breaks any floating/sidebar elements in Google Sheets. Just spent the past hour trying to figure out why I could not use Conditional Formatting.

**Comment by mminot-yseop** [comment]
Related: https://github.com/ClearURLs/Addon/issues/387

**Comment by VedantParanjape** [comment]
+1, same issue

**Comment by Cr8zyIvan** [comment]
Same issue here.

**Comment by siddharthsg2** [comment]
Same issue with me as well

**Comment by Tamarindo94** [comment]
Same, I'm kinda baffled that after years the extension still doesn't support exceptions, which would fix this and many other issues

**Comment by lasagna88** [comment]
Same issue, can't use for this reason

**Comment by KevinRoebert** [comment]
I believe I've identified the issue and applied a fix. Could you test it using version 1.27.3 to see if it resolves the problem?

Since this version isn't published yet, you'll need to install the add-on manually. You can download the ZIP file for manual installation here:

🔗 [Download ClearURLs v1.27.3](https://gitlab.com/ClearURLs/ClearUrls/-/jobs/artifacts/master/raw/ClearURLs.zip?job=bundle%20addon)

Let me know if this works for you or if any issues persist. Thanks for your help!

**Comment by Hermholtz** [comment]
Yes, the issue appears to be solved in v.1.27.3, it still occurs in 1.27.2. So please publish the new version to addons.mozilla.org. Thank you.

**Comment by KevinRoebert** [comment]
> Yes, the issue appears to be solved in v.1.27.3, it still occurs in 1.27.2. So please publish the new version to addons.mozilla.org. Thank you.

Version 1.27.3 has been already submitted to Mozilla (addons.mozilla.org) and Microsoft (https://microsoftedge.microsoft.com/addons) for review since yesterday. The reviews are normally done within one to two weeks.

**Comment by dreadbalaur** [comment]
Thanks. Is there a dummy guide on how to install it manually for firefox?

**Comment by Hermholtz** [comment]
1. You need Firefox Developer Edition. I use the Portable version so that it doesn't interfere with my normal browser - https://portableapps.com/apps/internet/firefox-developer-portable
2. Download and "Install" it. It just unpacks it in a subfolder, not really installing it.
3. Exit your main Firefox, launch the Developer Edition one.
4. Go to about:config, accept the warning.
5. In the filter box enter xpinstall.signatures.required, doubleclick the result to change the setting to false.
6. Download the addon ZIP file from https://github.com/ClearURLs/Addon/releases.
7. In the hamburger menu of the browser click "Add-ons and themes" (or select Tools -> Add-ons and themes from the menu).
8. Click the gear icon at the top-right, select "Install Add-on from file"
9. Point it at the downloaded ZIP file.
10. Enjoy!

You can try the same in normal Firefox version, but it won't allow you to install an unsigned extension from a ZIP file.

**Comment by KevinRoebert** [comment]
Version 1.27.3 is now also online at the addon store.

**Comment by dreadbalaur** [comment]
Thank you! Glad to hear. I will save the manual installation steps for future reference too.
